coalib/parsing/CliParsing.py

```diff
@@ -87,3 +87,31 @@ def parse_custom_settings(sections,
                                origin=origin,
                                section_name=key_touple[0],
                                from_cli=True)
+
+
+def check_conflicts(cli_sections, log_printer):
```

Comment:
Also - isn't there a better way to do this with the argparser itself ? This seems like a common issue that the argparser should handle.

I think maybe we shuold have a `parse_args()` function in DefaultArgParser which handels these checks.
http://stackoverflow.com/questions/4466197/python-argparse-with-dependencies

Basically, I see no reason why this check should be done after the sections are created. It should throw an error at the arg parsing.